docs/source/supergraphs.md

```diff
@@ -74,3 +74,16 @@ rover supergraph compose --config ./supergraph.yaml > prod-schema.graphql
 ```
 
 > For more on passing values via `stdout`, see [Using `stdout`](./conventions#using-stdout).
+
+#### Gateway compatibility
+
+The `rover supergraph compose` command produces a supergraph by utilizing the composition functions of the [`@apollo/federation`](https://www.apollographql.com/docs/federation/api/apollo-federation/) package.  As that library is still in pre-1.0 releases (as are Rover and Apollo Gateway), some updates to Rover may result in a supergraph with new functionality and therefore necessitate complimentary updates to Apollo Gateway.
```

Comment:
"produces a supergraph schema by using composition functions from the..."

"As that library" -> "Because that library"

"may result" -> "might result"

"in a supergraph" -> "in a supergraph schema with new functionality." <- end sentence

"and therefore necessitate..." -> "In turn, this might require corresponding updates to your gateway."